server/pension/tests.py

```diff
@@ -1,3 +1,8 @@
 from django.test import TestCase
 
-# Create your tests here.
+class TestPension(TestCase):
+    def test_index_view(self):
```

Comment:
@ordavidil - above which line (please provide the number, I'm not clear which one you mean)?